cpg_workflows/query_modules/seqr_loader_sv.py

```diff
@@ -441,19 +420,9 @@ def _filter_sample_cn(i, g):
         #     lambda g: g.new_call | hl.is_defined(g.prev_num_alt)
         # ),
         samples_no_call=_genotype_filter_samples(lambda g: g.num_alt == -1),
-        samples_num_alt=hl.struct(
-            **{
-                '%i' % i: _genotype_filter_samples(_filter_num_alt(i))
-                for i in range(1, 3, 1)
-            }
-        ),
+        samples_num_alt=hl.struct(**{'%i' % i: _genotype_filter_samples(_filter_num_alt(i)) for i in range(1, 3, 1)}),
```

Comment:
For these ones, I am extremely embarrassed at what my line length suggestion has done…